run.sh

```diff
@@ -2,6 +2,46 @@
 
 set -o errexit
 
+usage() {
+    echo "deploy                                         -- Deploy SUSE Containerized OpenStack."
+    echo "update_openstack                               -- Update OpenStack deployment."
+    echo "add_openstack_compute                          -- Add OpenStack compute node. User need to provide compute node name in inventory file."
```

Comment:
need->needs?   "add compute node host(s) in the inventory file" -- nits

Reply:
will fix it.